src/components/Page/PageFooter.vue

```diff
@@ -1,24 +1,26 @@
 <template>
     <div :class="['footer', this.$root.tabTheme.rendered, $localData.settings.pixelScaling ? 'pixelated' : false ]" :style="styleObject">
         <Media :url="bannerImage[0]" class="bannerImage left" draggable="false" />
+        <Ad v-if="show_ad" />
```

Comment:
Lifted the sbahj tbiy ad into its own component and set up logic for displaying it in the pagefooter (where ads were historically positioned) by a mod